packages/office-ui-fabric-react/src/components/Nav/Nav.Props.ts

```diff
@@ -91,16 +91,6 @@ export interface INavLinkGroup {
    * The name to use for functional automation tests
    */
   automationId?: string;
-
-  /**
-   * If true, the group should render collapsed by default
-   */
-  collapseByDefault?: boolean;
```

Comment:
This is also due to merge complications. I should have double-checked before I created the new PR.  Sorry.